frameworks/hdfs/universe/config.json

```diff
@@ -83,14 +83,20 @@
               "type": "boolean",
               "default": false
             },
-            "keytabs_uri": {
+            "kdc_host_name": {
               "type": "string",
-              "description": "The URI from which Keytabs tar.gz can be downloaded. Only relevant when secure_mode is enabled.",
-              "default": ""
+              "description": "The name or address of a host running a KDC for the realm.",
+              "default": "kdc.marathon.l4lb.thisdcos.directory"
             },
-            "krb5_conf_uri": {
+            "kdc_host_port": {
               "type": "string",
-              "description": "The URI of the krb5.conf file.  This file will be downloaded and used by each HDFS task."
+              "description": "The port of the host running a KDC for that realm.",
+              "default": "88"
+            },
+            "keytab_secret_path": {
```

Comment:
For the service account secret, we just use `service_account_secret`. Is this also a path, or is there something different here?

Reply:
Ah in that case, I'll drop the `_path` suffix, though it could be more clear from a UX perspective that `service_account_secret` is a field expecting a path to a secret. From an initial impression, it could be interpreted as that field wanting the secret itself, and not the path.